### ISSUE TITLE
[Editor] Syntax highlight with template strings

### ISSUE THREAD

**Issue by jasonLaster** [devtools-html/debugger.html#2944]
Syntax highlighting breaks when there's a template string. We should see if this is fixed in a codemirror update

<img src='https://shipusercontent.com/bc139669cf9906f87655274d5650a785/Screen%20Shot%202017-05-17%20at%205.09.32%20PM.png' title='Screen Shot 2017-05-17 at 5.09.32 PM.png' width=642>

**Comment by peterschussheim** [comment]
@jasonLaster Is the example in the screenshot in the debugger-example repo?  I am trying a few examples I just wrote and haven't been able to reproduce the broken highlighting.

**Comment by jasonLaster** [comment]
It's coming from debugging the debugger. Let's add it to an example

**Comment by peterschussheim** [comment]
### tl;dr
I haven't been able to reproduce the bug in your screenshot.

---

@jasonLaster Wrapping up some template string examples for a PR tomorrow.  I've made two recordings (**warning:** these were too large to embed as gif) in an attempt to reproduce the bug in your screenshot, but still unable.  Perhaps you have a setting enabled that I didn't have?

## Recordings
### Example 1:

[Chrome Canary debugging](http://recordit.co/6sbnnkz7bX) `debugger.html` which is connected to Firefox client that is debugging another instance of FF:

---

### Example 2:
[Firefox debugging](http://recordit.co/ZnPsIea7Zx) `debugger.html` which is connected to Firefox client that is debugging another instance of FF.

**Comment by darkwing** [comment]
I cannot reproduce this either

**Comment by jasonLaster** [comment]
Thanks!

**Comment by jasonLaster** [comment]
yep - all fixed!

**Comment by digitarald** [comment]
Just saw this on Nightly:

<img width="572" alt="image" src="https://user-images.githubusercontent.com/8599/33579054-4ed62bea-d8fd-11e7-887a-a556fca69f53.png">

https://perf-html.io/public/bf377b229ec75f94bf3596e45d7bff52a9203f74/calltree/?hiddenThreads=&thread=3&threadOrder=0-2-3-4-5-1&v=2
webpack:///src/actions/receive-profile.js line 451

@darkwing can you reproduce?

**Comment by jasonLaster** [comment]
i think this is fixed
